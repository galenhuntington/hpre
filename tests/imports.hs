import Data.List as List
import qualified Control.Monad as Monad

--+
import Data.Char
import Data.String as String, as S
import Data.Ord, as Ord
import Data.List, -- comment
   as List

-- ignored
--+
