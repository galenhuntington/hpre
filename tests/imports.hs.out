import Data.List as List
import qualified Control.Monad as Monad


import Data.Char 
import qualified Data.String  as String;import qualified Data.String  as S
import Data.Ord ;import qualified Data.Ord  as Ord
import Data.List ;import qualified Data.List 
   as List

-- ignored

